src/cyw43_ll.c

```diff
@@ -1183,7 +1183,7 @@ void cyw43_ll_process_packets(cyw43_ll_t *self_in) {
         } else if (ret == DATA_HEADER) {
             cyw43_cb_process_ethernet(self->cb_data, len >> 31, len & 0x7fffffff, buf);
         } else {
-            CYW43_WARN("got unexpected packet %d\n", ret);
+            CYW43_VDEBUG("got unexpected packet %d\n", ret);
```

Comment:
I think it should only be a VDEBUG print if `ret == -9` (which could probably do with a macro define for that constant) and if `CYW43_USE_SPI` is enabled.